pkg/kubectl/cmd/testing/fake.go

```diff
@@ -680,6 +685,7 @@ func testDynamicResources() []*discovery.APIGroupResources {
 			VersionedResources: map[string][]metav1.APIResource{
 				"v1beta1": {
 					{Name: "deployments", Namespaced: true, Kind: "Deployment"},
+					{Name: "thirdpartyresources", Namespaced: false, Kind: "ThirdPartyResource"},
```

Comment:
Ok. Do you want me to abandon this ? or address the final reviews and bring it to a mergeable state ?